src/Mod/PartDesign/Gui/Command.cpp

```diff
@@ -1633,16 +1633,16 @@ bool dressupGetSelected(Gui::Command* cmd, const std::string& which,
 
     if (selection.size() == 0) {
         QMessageBox::warning(Gui::getMainWindow(), QObject::tr("Wrong selection"),
-            QObject::tr("Select an edge, face or body."));
+            QObject::tr("Select an edge, face, or body."));
```

Comment:
@yorikvanhavre https://www.businessinsider.com/always-use-the-oxford-comma-2015-5

Reply:
@yorikvanhavre it's splitting hairs and somewhat stylistic (although, IMHO grammatically it makes more sense) but for the record I've added this modification to other strings before in the source. So this would be more of a uniformity fix.

Reply:
* https://github.com/FreeCAD/FreeCAD/blame/83f34a1e5b11ee900ddaf92e0f0fd1afc1c7595c/src/Mod/Arch/Resources/ui/preferences-ifc.ui#L434
* https://github.com/FreeCAD/FreeCAD/blob/master/src/Mod/Arch/ArchCommands.py#L640
* https://github.com/FreeCAD/FreeCAD/blob/master/src/Mod/Draft/draftutils/groups.py#L59 (this was vocx actually)

Reply:
@yorikvanhavre ping

Reply:
Please merge. I'd like to trigger another crowdin update after this.

Reply:
Okay I'm convinced :laughing: